app/services/adapters/structs/applicant.rb

```diff
@@ -12,7 +12,8 @@ def correspondence_address
       def serializable_hash(options = {})
         super(
           options.merge(
-            methods: []
+            methods: [],
+            except: [:relationship_to_someone_else, :residence_type]
```

Comment:
Is this meant to be in this PR?

Reply:
I've just merged my pr so you should be able to get rid of this